### ISSUE TITLE
Why delete the sigmoid in the output layers?

### ISSUE THREAD

**Issue by tongxueqing** [fnzhan/Illumination-Estimation#18]
Thanks for your wonderful work. I wonder why you delete  the sigmoid in the output layers in DenseNet.py in the latter version? And I guess it will be  reasonable if self.fc_dist(out) is followed by softmax , since the sum of gt_distribution is one .(https://github.com/fnzhan/Illumination-Estimation/blob/master/RegressionNetwork/DenseNet.py)

**Comment by fnzhan** [comment]
Hi, I just find including sigmoid will make the network more difficult to converge during training.

**Comment by tongxueqing** [comment]
Thanks for your timely reply. Can you share how large is the subset when you begin to add schedule for learning rate? And I found you do not save ambient term in the test.py, is it due to the ambient term is not important?

**Comment by tongxueqing** [comment]
https://github.com/fnzhan/Illumination-Estimation/blob/master/RegressionNetwork/DenseNet.py
The if condition does not hold, it is a bug or you intend to do so?
<img width="662" alt="截屏2022-07-29 下午3 21 58" src="https://user-images.githubusercontent.com/49693188/181706018-d52a3177-015e-4ab6-b0cc-f03856dd1e15.png">
